app/editor/src/features/admin/reports/ReportForm.tsx

```diff
@@ -50,7 +50,7 @@ const ReportForm: React.FC = () => {
 
   const [report, setReport] = React.useState<IReportModel>({
     ...defaultReport,
-    ownerId: userInfo?.id ?? 0,
+    ownerId: userInfo?.id,
```

Comment:
Default to `undefined`.  On a page refresh it was getting set to `0` which was an invalid foreign key.